inc/spbc-tools.php

```diff
@@ -484,3 +484,325 @@ function spbc_get_custom_scanner_launch_data($first_start = false, $settings = a
         'start_time' => $start_time
     );
 }
+
+/**
+ * Enqueue JS scripts, css and localization for widget.
+ * @return void
+ */
+function spbc_widget_scripts_init()
+{
+    global $spbc;
+    wp_enqueue_script(
+        'spbc-widget-chart-js',
+        SPBC_PATH . '/js/lib/chart/spbc-dashboard-widget--chartjs.min.js',
+        array('jquery'),
+        SPBC_VERSION,
+        false
+    );
+    wp_enqueue_script(
+        'spbc-widget-dashboard',
+        SPBC_PATH . '/js/spbc-dashboard-widget.min.js',
+        array('spbc-widget-chart-js'),
+        SPBC_VERSION,
+        false
+    );
+    wp_enqueue_style(
+        'spbc_admin_css_widget_dashboard',
+        SPBC_PATH . '/css/spbc-dashboard-widget.min.css',
+        array(),
+        SPBC_VERSION,
+        'all'
+    );
+
+    $brief_data = isset($spbc->data['brief_data']) ? $spbc->data['brief_data'] : array();
+    $bfp_data = !empty($brief_data['bfp_data']) ? $brief_data['bfp_data'] : array();
+    $fw_data = !empty($brief_data['bfp_data']) ? $brief_data['fw_data'] : array();
+
+    sort($bfp_data);
+    sort($fw_data);
+
+    wp_localize_script('spbc-widget-dashboard', 'spbcDashboardWidget', array(
+        'data_bfp' => $bfp_data,
+        'data_fw' => $fw_data,
+    ));
+}
+
+/**
+ * Set brief data for widget to the State.
+ * @param bool $direct_call if direct call (from the frontend)
+ * @return void
+ */
+function spbc_set_brief_data($direct_call = false)
```

Comment:
`$direct_call` parameter is redundant here because it always true.

Reply:
Thanks, done.